bazel/repository_locations.bzl

```diff
@@ -191,4 +191,9 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "subpar-1.3.0",
         urls = ["https://github.com/google/subpar/archive/1.3.0.tar.gz"],
     ),
+    com_google_quiche = dict(
```

Comment:
Should this be `com_googlesource_quiche` to reflect the convention?

Reply:
done